docs/developer-guide/triggers/webhook.md

```diff
@@ -1,14 +1,17 @@
+---
+order: 2
+---
 
 # Webhook
 
 ```yaml
 type: "io.kestra.core.models.triggers.types.Webhook"
 ```
 
-> A Webbook trigger allows you to trigger a flow from a webhook URL.
-> The trigger will generate a key that must be used on the URL : `/api/v1/executions/webhook/{namespace}/[flowId]/{key}`.
-> Kestra accepts `GET`, `POST` & `PUT` requests on this URL.
-> The whole body & headers will be available as a variables.
+> A Webhook trigger allows to trigger a flow from a webhook URL.
+> At trigger creation time, Kestra will generate a key that must be used on the URL that triggers the flow: `/api/v1/executions/webhook/{namespace}/[flowId]/{key}`.
+> Kestra accepts `GET`, `POST` and `PUT` requests on this URL.
+> The whole request body and headers will be available as a variables.
```

Comment:
As one variable or as separate variables?

Reply:
Also, can we remove the quotation section since we aren't quoting anything here?

Reply:
> As one variable or as separate variables

Separate

> Also, can we remove the quotation section since we aren't quoting anything here?

If you refere to the quotting formatting `>` it's used everywhere so I prefere to keep it, we can revisit documentation design later as I too found it strange to use quote everywhere.